### DISCUSSION TITLE
cockpit-storage no output (endless rotating clock)

### DISCUSSION THREAD

**Discussion by mibenten** [cockpit-project/cockpit#18401] [Q&A] [answered]
ubuntu 22.04, hp microserver gen10, software raid

dpkg -l | grep cock
ii  cockpit                               285-1~bpo22.04.1                        all          Web Console for Linux servers
ii  cockpit-bridge                        285-1~bpo22.04.1                        amd64        Cockpit bridge server-side component
ii  cockpit-networkmanager                285-1~bpo22.04.1                        all          Cockpit user interface for networking
ii  cockpit-packagekit                    285-1~bpo22.04.1                        all          Cockpit user interface for packages
ii  cockpit-pcp                           285-1~bpo22.04.1                        amd64        Cockpit PCP integration
ii  cockpit-storaged                      285-1~bpo22.04.1                        all          Cockpit user interface for storage
ii  cockpit-system                        285-1~bpo22.04.1                        all          Cockpit admin interface for a system
ii  cockpit-ws                            285-1~bpo22.04.1                        amd64        Cockpit Web Service

js-console on chrome give me:

updates.js:2 Tracer failed: "{"problem":null,"exit_status":1,"exit_signal":null,"message":"Traceback (most recent call last):\n  File \"<string>\", line 2, in <module>\nModuleNotFoundError: No module named 'tracer'"}", data: """"
(anonymous) @ updates.js:2
(anonymous) @ cockpit.js:1
(anonymous) @ cockpit.js:1
L @ cockpit.js:1
setTimeout (async)
A @ cockpit.js:1
N @ cockpit.js:1
M @ cockpit.js:1
P.e.resolve @ cockpit.js:1
d @ cockpit.js:1
(anonymous) @ cockpit.js:1
(anonymous) @ cockpit.js:1
L @ cockpit.js:1
setTimeout (async)
A @ cockpit.js:1
m @ cockpit.js:1
c @ cockpit.js:1
value @ cockpit.js:1
f @ cockpit.js:1
(anonymous) @ cockpit.js:1
B.n.onmessage.e.dispatch_data @ cockpit.js:1
(anonymous) @ cockpit.js:1
postMessage (async)
(anonymous) @ index.js:2
B.n.onmessage.e.dispatch_data @ cockpit.js:1
storage.js:2 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'v')
    at I_ (storage.js:2:1188478)
    at A_ (storage.js:2:1189266)
    at storage.js:2:1220795
    at $_.update (storage.js:2:1220814)
    at storage.js:2:1215173

**Comment by martinpitt** [comment, marked as answer]
Decoding the uglified stack trace: I_() is pkg/storaged/fsys-tab.jsx get_fstab_config(), and the calling function A_() is check_mismounted_fsys(). So apparently one of the type/dir/opts fields of `Configuration` is "undefined".

@mibenten, can you please copy&paste the contents of your /etc/fstab and /etc/crypttab here? Perhaps we can better understand what happens here.

**Reply by martinpitt** [reply]
This is an actual bug, so let's rather track this in issue #18406.
